subsys_tello_sensors.py

```diff
@@ -9,11 +9,36 @@
 from typing import Union
 
 
-class DroneStatus:
-    battery = 0
-    roll = 0
-    pitch = 0
-    yaw = 0
+class DroneState:
+    """
+    Storage class containing every Tello internal variables that are retrieved with the
+    Tello.get_current_state() method.
+
+    In cls.update_state(variables_dict), a class attribute is created (or updated if it already exists)
+    for each field of the input dictionary variables_dict.
+    The class attribute gets the name of the dict key and the value of its corresponding
+    dict element
+
+    The cls.__getDict__() method is used to retrieve only the variables to print in the pygame display window.
+    """
+    @classmethod
+    def update_state(cls, variables_dict: dict):
```

Comment:
I think you shouldn't pass this as a variable, just execute TELLO.get_current_state() from here and directly use the returned variables => the update_state() shouldn't require any varariable